openmc/lib/core.py

```diff
@@ -186,15 +186,15 @@ def init(args=None, intracomm=None):
     if args is not None:
         args = ['openmc'] + list(args)
         argc = len(args)
-
-        # Create the argv array. Note that it is actually expected to be of
-        # length argc + 1 with the final item being a null pointer.
-        argv = (POINTER(c_char) * (argc + 1))()
-        for i, arg in enumerate(args):
-            argv[i] = create_string_buffer(arg.encode())
     else:
-        argc = 0
-        argv = None
+        args = ['openmc']
+        argc = 1
```

Comment:
You could use `argc = len(args)` outside of the `if` to handle both cases

Reply:
Ah yeah, that would be better.